### PR TITLE
Add /16-04/azure page and contact-us

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Created the new /16-04/azure page and modal contact us

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/16-04/azure and http://0.0.0.0:8001/16-04/azure#get-in-touch
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the copy docs - [page](https://docs.google.com/document/d/1V99KwimN8Hla8L_CikNc6I0amqj99cSrBsVlAjIkBqE/edit#), [contact-form](https://docs.google.com/document/d/1NgT2MPNmF1SdVMCGU-Vr_I2wlm25C5z-p_rPZTnJrww/edit#)

## Issue / Card

Fixes [#4025](https://github.com/canonical-web-and-design/web-squad/issues/4025)

## Screenshots

image

### CODE REVIEW

**Comment by anasereijo** [comment]
Hi @pmahnke just some small changes:

Screenshot 2021-04-15 at 10 38 23

Screenshot 2021-04-15 at 10 40 53

I also asked @lechsandecki about which logo to use for Azure

File: templates/16-04/azure.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      <h3 class="p-muted-heading">
+        Upgrading existing instances
+      </h3>
+      <p>
+        If you are looking to upgrade your existing Ubuntu 16.04 LTS Azure instances you can <a class="p-link--external" href="https://azuremarketplace.microsoft.com/uk-ua/marketplace/apps/Canonical.UbuntuAdvantageSupport?tab=Overview">enable the Ubuntu Advantage subscription directly from the Azure marketplace</a>.
+      </p>
+      <p>
+        Please note that this will enable Ubuntu Advantage for all Ubuntu instances under the selected subscription.
+      </p>
+    </div>
```
Comment (bethcollins92) [inline templates/16-04/azure.html:107]: Missing paragraph here "If you are looking to upgrade your existing...."

File: templates/16-04/azure.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      </p>
+      <p>
+        <a class="js-invoke-modal" shref="/azure/contact-us">
+          Contact us with your migration questions&nbsp;&rsaquo;
+        </a>
+      </p>
+    </div>
+  </div>
+</section>
+
```
Comment (bethcollins92) [inline templates/16-04/azure.html:128]: Lech replied in the copy doc about the image so not sure if you want to add it here?

File: templates/shared/forms/interactive/azure-1604.html [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+              <input type="checkbox" id="about-your-1604-instances-private-or-edge-cloud">
+              <label for="about-your-1604-instances-private-or-edge-cloud">Private or edge cloud</label>
+            </div>
+            <div class="col-4 u-sv3">
+              <input type="checkbox" id="about-your-1604-instances-development-and-ci/cd">
+              <label for="about-your-1604-instances-development-and-ci/cd">Development and CI/CD</label>
+              <input type="checkbox" id="about-your-1604-instances-standard-production">
+              <label for="about-your-1604-instances-standard-production">Standard Production</label>
+              <input type="checkbox" id="about-your-1604-instances-mission-critical-production-regulated-workloads">
+              <label for="about-your-1604-instances-mission-critical-production-regulated-workloads">Mission-critical Production Regulated workloads</label>
```
Comment (bethcollins92) [inline templates/shared/forms/interactive/azure-1604.html:38]: This is split up into two checkboxes on the copy doc "Mission-critical Production" and "Regulated workloads" - is putting it in one intentional?

File: templates/shared/forms/interactive/azure-1604.html [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+            </div>
+          </div>
+        </div>
+
+        <div class="js-formfield">
+          <h3 class="p-heading--five">Which of the following topics are important to you?</h3>
```
Comment (bethcollins92) [inline templates/shared/forms/interactive/azure-1604.html:45]: I think this section would look more even if it was split into 3 cols of 6 checkboxes each. (It's fine as it is though)

**Review by bethcollins92 — APPROVED** [review]
LGTM with a couple of questions/suggestions inline 👍🏼

File: templates/16-04/azure.html (hunk 1 above)
Comment (pmahnke) [reply templates/16-04/azure.html:107]: isn't that on line 102?

File: templates/16-04/azure.html (hunk 2 above)
Comment (pmahnke) [reply templates/16-04/azure.html:128]: Done, I think it is stupid, but done.

**Comment by anasereijo** [comment]
@pmahnke perfeito! 👍

File: templates/16-04/azure.html (hunk 1 above)
Comment (bethcollins92) [reply templates/16-04/azure.html:107]: ahh yes sorry missed that!

**Pull request merged**